src/tests/compile.js

```diff
@@ -101,11 +104,11 @@ async function compileInkFile(inklecate) {
   if (errors.length === 0) {
     console.log("Done.");
   } else {
-    errors.forEach((error) => console.error(`\n${error.reason.message}`));
+    errors.forEach((error) => console.error(`\n${error.reason.message.substring(0,255)}`));
```

Comment:
When using the minified "inkjs-compiler.js", error message include the full unreadable stack trace. 255 chars should be enough to identify which test failed and run it individually.